roles/addons/clone/files/cloneset.py

```diff
@@ -85,7 +89,8 @@ def set_default_boot(node, boot, diskless_image=None):
             if 'network' in passed_arguments.force:
                 dedicated_parameters = str('ip='+nodes_parameters[str(node)]["network"]["node_main_network_interface_ip"]+'::'+nodes_parameters[str(node)]["network"]["node_main_network_gateway"]+':'+nodes_parameters[str(node)]["network"]["node_main_network_netmask"]+':'+str(node)+':'+nodes_parameters[str(node)]["network"]["node_main_network_interface"]+':none')
 
-            generic_node_ipxe = '\n'.join(('#!ipxe',
+            generic_node_ipxe = '\n'.join((
+                '#!ipxe',
```

Comment:
We need to ensure #!ipxe is still at the very beginning of the file here. If not, iPXE will fail :-|